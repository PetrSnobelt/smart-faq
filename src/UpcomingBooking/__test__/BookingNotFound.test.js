// @flow

import * as React from 'react';
import { shallow } from 'enzyme';

import BookingNotFound from '../BookingNotFound';

describe('BookingNotFound', () => {
  const component = shallow(<BookingNotFound />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
