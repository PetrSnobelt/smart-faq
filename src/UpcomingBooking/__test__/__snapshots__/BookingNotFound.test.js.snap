// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookingNotFound should match snapshot 1`] = `
<div
  className="jsx-1169529336 UpcomingBooking"
>
  <div
    className="jsx-1169529336 feedback"
  >
    <Typography
      size="large"
      type="secondary"
    >
      No Bookings found
    </Typography>
  </div>
  <JSXStyle
    css=".UpcomingBooking.jsx-1169529336{width:480px;padding:40px;padding-top:128px;background-color:#f5f7f9;height:100%;}div.feedback.jsx-1169529336{margin-top:32px;text-align:center;}"
    styleId="1169529336"
  />
</div>
`;
